genezio.yaml

```diff
@@ -20,7 +20,3 @@ frontend:
     deploy:
       - npm install @genezio-sdk/${{projectName}}@1.0.0-${{stage}}
       - npm install
-    # List of scripts to run when starting the local development server.
-    start:
-      - npm install --silent
-      - npm run start
```

Comment:
This will disable starting the frontend at `genezio local`. Do we want this?

I noticed the other frontend templates still have `start` in place.

Reply:
Nevermind, I read the context from slack